docs/user-manual/modules/faq/pages/what-platforms-are-supported.adoc

```diff
@@ -1,5 +1,7 @@
 = What platforms are supported?
 
 - Camel 2.x supports JDK 8
-- Camel 3.x supports JDK 8 (secondary) and 11 (primary)
+- Camel 3.x supports JDK 8 up to its 3.14.x branch
+- Camel 3.x supports JDK 11
```

Comment:
3.x supports JDK 17 from 3.18 onwards.

Reply:
According to your table, it's 3.17, but 3.17 was not LTS.
What does that means, that you started testing with Java-17 back then? because I don't see any relevant change regarding java-17 in the pom.